src/main/java/org/prnhs/javaee/swim/core/CoreConfig.java

```diff
@@ -0,0 +1,14 @@
+package org.prnhs.javaee.swim.core;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@Configuration
+@EnableAutoConfiguration
+@EnableJpaRepositories(basePackages = "org.prnhs.javaee.swim.dao")
```

Comment:
enable jpa repositories and point spring to the package that will contain said daos.  Placing a dao in another package will not get picked up by spring